README.md

```diff
@@ -185,6 +185,15 @@ compile('com.rollbar:rollbar-java:1.0.1')
 compile('com.rollbar:rollbar-android:1.0.1@aar')
 ```
 
+## How payloads are sent
+
+The actual notifier configuration builds a notifier that uses a BufferedSender to send the items
+to Rollbar. That sender is built using an unbound memory queue and a scheduled thread to send
+the events from the queue. 
+The queue and as well the frequency of the scheduled thread can be customize
```

Comment:
customize => customized

Reply:
`The queue and` => the `and` should be removed

Reply:
Thanks!